packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -2,114 +2,127 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:flutter/material.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/rendering.dart';
-
 import 'package:flutter/cupertino.dart';
 
-const EdgeInsets _kHorizontalItemPadding =
-    const EdgeInsets.symmetric(horizontal: 16.0, vertical: 0.0);
+//Minimal padding from horizontal edges of segmented control to edges of encompassing widget.
```

Comment:
Comments should have a space after the "//" (and below).
Also:
Minimal --> Miniumum